scripts/common/rook.sh

```diff
@@ -133,9 +138,15 @@ function remove_rook_ceph() {
         kubectl -n kurl scale deploy ekc-operator --replicas=1
     fi
 
+    rm -rf /var/lib/rook || true
+    rm -rf /opt/replicated/rook || true
```

Comment:
I think this looks good but I'd like @laverya 's opinion as to whether he feels it is safe to remove the data here if potentially other operations failed above with only a logWarn

Reply:
From the docs; https://rook.io/docs/rook/v1.10/Getting-Started/ceph-teardown/#delete-the-operator-and-related-resources

<img width="811" alt="Screenshot 2023-03-20 at 21 58 05" src="https://user-images.githubusercontent.com/7708031/226474765-5a0c1169-4b7e-4466-8775-985939ec0ff2.png">

Also, @laverya comment that would make sense we remove here: https://app.shortcut.com/replicated/story/70238/migration-from-rook-does-not-clean-the-data-stored-and-are-not-following-the-rook-teardown-docs#:~:text=Mar%2010%202023%2C%205%3A11%20pm

Reply:
yes. im just worried that there are plenty of conditionals above here that do not return when a command fails and instead log warn and continue. is it safe to delete the directories if we have not yet successfully removed rook and ceph?

Reply:
I think to get here the namespace has to have been successfully deleted, so it's fine to delete the hostpaths

Reply:
We only are in the place where we delete the data after:
a) we migrate 
b) we delete the namespace

The path : https://github.com/replicatedhq/kURL/pull/4223/files#diff-d1a3d4d46c348ae45d17405ff5db5f2a57f6de8264635200a462fb8a7ac9b7bdR91-R93

Will also allow the Operator remove the data. 
That seems sorted out the scenarios to remove Rook for upper versions either.

Reply:
sounds good. approving